pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[project]
+name = "pykeepass"
+readme = "README.rst"
+description = "Python library to interact with keepass databases (supports KDBX3 and KDBX4)"
+authors = [
+    { name = "Philipp Schmitt", email = "philipp@schmitt.co" }
+]
+license = {text = "GPL-3.0"}
+keywords = ["valorant", "vault", "keepass"]

```

Comment:
What is valorant?

Reply:
Nope, definitely have to delete that. Small mistake from using a template.